draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -108,58 +108,96 @@ contributor:
+     target: https://datatracker.ietf.org/doc/draft-ietf-ccamp-rfc9093-bis/
+
+   ietf-optical-interface-yang:
+     title: "A YANG model to manage the optical interface parameters for an external transponder in a WDM network"
+     date: 2025-01-06 
+     target: https://datatracker.ietf.org/doc/draft-ietf-ccamp-dwdm-if-param-yang/
+
+  
 
 --- abstract
 
-This draft outlines the modeling of optical pluggables within a host packet device in the context of a packet over optical network. The model encompasses all pertinent properties of the pluggable for various packet over optical use cases and is partitioned into three primary areas: the optical media interface, the electrical plug to host interconnect, and the physical equipment of the pluggable.
+This draft outlines the modeling of optical pluggables within a host packet device in the context of a packet over optical network. The model encompasses all pertinent properties of the pluggable for various packet over optical use cases and is partitioned into three primary areas: the optical media side, the electrical plug to host interconnect, and the physical equipment of the pluggable.
 
 Included in the model are representations of configuration, states, and telemetry data, as well as of profiles and coherent plug capabilities. Emphasizing the importance of considering both vendor-agnostic and vendor-specific attributes in modeling coherent pluggables.
 
-Drawing from existing models in IETF, OpenConfig, ITU-T, OIF, and TAPI, this model offers enhanced uniform structuring and naming. Additionally, it provides insight into the mapping to plug interface structures defined in CMIS.
+Drawing from existing IETF models and potentially complementing that with input from other standard or industrial forum models (ITU-T, OpenConfig , ONF TAPI etc.) , this model offers enhanced uniform structuring and naming.
 
-This draft introduces the concpet of "Coherent Pluggable Manifest", where a represents the capabilities of a type&version of pluggable, maintained in a public repository of all pluggables. This document also covers gap analysis of current IETF drafts and other SDOs on coherent pluggables attributes and provides the complete lifecycle of a coherent pluggable from operators' approval through viability assessment to deployment.
+This draft introduces the concept of "Coherent Pluggable Manifest", where it represents the capabilities of pluggable, maintained in a repository. This document also covers gap analysis of current IETF drafts and other SDOs on coherent Pluggable attributes and provides the complete lifecycle of a coherent pluggable from operators' approval through viability assessment to deployment.
+
+This document also covers an analysis of the gap between current IETF drafts and the corresponding works in other standards bodies and industry. The lifecycle of a coherent pluggable including operators' approval, viability assessment to deployment and deployment is also covered.
```

Comment:
nothing is changed and the repetition is still present.

Reply:
It seems to me that this part "This document also covers an analysis of the gap between current IETF drafts and the corresponding works in other standards bodies and industry. The lifecycle of a coherent pluggable including operators' approval, viability assessment to deployment and deployment is also covered." is already reported before and it seems a repetition with other words. Am I missed something?

Reply:
this sentence outlines two aspects:
1- this talk about life cycle of coherent Pluggables
2- Covering Gap analysis
Both are not discussed in previous sentences.